airflow/cli/commands/task_command.py

```diff
@@ -326,7 +326,7 @@ def _move_task_handlers_to_root(ti: TaskInstance) -> Generator[None, None, None]
     console_handler = next((h for h in root_logger.handlers if h.name == "console"), None)
     with LoggerMutationHelper(root_logger), LoggerMutationHelper(ti.log) as task_helper:
         task_helper.move(root_logger)
-        if IS_K8S_EXECUTOR_POD:
+        if IS_K8S_EXECUTOR_POD or IS_EXECUTOR_CONTAINER:
```

Comment:
Nittiest of nitpicks, non-blocking thought:  in two places you use `if IS_K8S_EXECUTOR_POD or IS_EXECUTOR_CONTAINER:`, I wonder if a calculated constant somewhere like `IS_CONTAINERIZED = any(IS_K8S_EXECUTOR_POD, IS_EXECUTOR_CONTAINER)`... but that's likely pointless overkill right now, but if/when other providers add their own containerized executors, that may be needed...

Reply:
From another direction, it seems that `IS_EXECUTOR_CONTAINER` is never used on its own, which makes me feel perhaps only the `IS_CONTAINERIZED` constant is needed, and `IS_EXECUTOR_CONTAINER` can be removed entirely. The same applies for the test parameters.

Reply:
Sorry folks (@ferruzzi @uranusjr), I thought I put enough details in the description but perhaps I should have added more!

This new constant will be a generalized replacement for the K8s constant we have now. Any new containerized executor should use this one. Which our new ECS executor does use (link in description). I did not remove the K8s environment variable in this PR to keep the scope small. This change is _just_ the core pieces. The K8s executor will be migrated over to this env var in a follow up PR.